src/@polymer/lit-element-decorators.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference types="reflect-metadata" />
+
+/**
+ * Options for the @property decorator.
+ * See https://www.polymer-project.org/2.0/docs/devguide/properties.
+ */
+export interface PropertyOptions {
+  /**
+   * This field can be omitted if the Metadata Reflection API is configured.
+   */
+  type?: BooleanConstructor|DateConstructor|NumberConstructor|StringConstructor|ArrayConstructor|ObjectConstructor;
+  notify?: boolean;
+  reflectToAttribute?: boolean;
+  readOnly?: boolean;
+  computed?: string;
+  observer?: string|((val: any, old: any) => void);
+}
+
+export function customElement(tagname: string) {
+  return (clazz: any) => {
+    window.customElements.define(tagname!, clazz);
+  };
+}
+
+export function property(options?: PropertyOptions) {
+  return (prototype: any, propertyName: string): any => {
+    options = options || {};
+    options.type = options.type || reflectType(prototype, propertyName);
+    createProperty(prototype, propertyName, options);
+  };
+}
+
+function createProperty(prototype: any, propertyName: string, options: PropertyOptions = {}): void {
+  if (!prototype.constructor.hasOwnProperty('properties')) {
+    Object.defineProperty(prototype.constructor, 'properties', { value: {} });
+  }
+  prototype.constructor.properties[propertyName] = options;
+}
+
+function reflectType(prototype: any, propertyName: string): any {
+  const { hasMetadata = () => false, getMetadata = () => null } = Reflect;
```

Comment:
Give how large The Reflect polyfill is and how the metadata API is no longer even speculative, I'd prefer to remove support altogether.

Reply:
ok, done